### DISCUSSION TITLE
show private contributions

### DISCUSSION THREAD

**Discussion by dingrobu** [anuraghazra/github-readme-stats#3366] [Q&A] [answered]
### Discussed in https://github.com/anuraghazra/github-readme-stats/discussions/3321

<div type='discussions-op-text'>

<sup>Originally posted by **Aydinhamedi** October  3, 2023</sup>
### Is your feature request related to a problem? Please describe.

No.

### Describe the solution you'd like

I want the card to show the privet contributions too.

### Additional context

_No response_</div>

**Comment by dingrobu** [comment]
Hi all

I managed to deploy my own instance on vercel. However I am not seeing my private contributions using this code chunk (of course with my user name and my own app link):

`[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)`

What could be the issue with this?

The domains generated by vercel link to the repository https://github.com/anuraghazra/github-readme-stats. 

Could this be the issue?

Thanks in advance

**Reply by qwerty541** [reply, marked as answer]
Hey, @dennisgrobe! Did you set correct permissions for your PAT (Personal Access Token (GitHub API key))? Make sure that you enabled `repo` and `user` permissions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)

**Reply by lsan31296** [reply]
Changed the PAT settings after deploying on Vercel, still not seeing the private contributions on the GitHub statistics card. I do have the contribution settings set to be visible, I'm not sure what is missing.

**Reply by karloows** [reply]
@lsan31296 did you redeploy?

**Reply by qwerty541** [reply]
> Changed the PAT settings after deploying on Vercel, still not seeing the private contributions on the GitHub statistics card. I do have the contribution settings set to be visible, I'm not sure what is missing.

Hello @lsan31296 

In case you actually properly configured the personal access token permissions then your private contributions should be already included into the stats. Note that the screenshot above displays the permissions only for the classic token. In case you are using the fine-grained token then you need to apply different configurations. Here is a link on our recently updated documentation about configured both types of PATs https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#first-step-get-your-personal-access-token-pat

**Reply by lsan31296** [reply]
Hey, I believe my issue was I did not redeploy my vercel app with the new PAT token (a classic token). Once I redeployed and used the new URL it seemed to work. Thank you.
